ui/apps/platform/src/Containers/NetworkGraph/components/ClusterSelector.tsx

```diff
@@ -37,15 +37,20 @@ function ClusterSelector({
     const clusterSelectOptions: JSX.Element[] = clusters.map((cluster) => (
         <SelectOption key={cluster.id} value={cluster.name}>
             <span>
-                <ClusterIcon /> {cluster.name}
+                <ClusterIcon className="pf-u-mr-xs" /> {cluster.name}
             </span>
         </SelectOption>
     ));
 
     return (
         <Select
             isPlain
-            placeholderText={<em>Select a cluster</em>}
+            placeholderText={
+                <span>
+                    <ClusterIcon className="pf-u-mr-xs" />{' '}
```

Comment:
i've noticed that prettier sometimes adds this to the end of lines in markup -- does anyone know if it's necessary?

Reply:
It has something to do with the way React translates the JSX to HTML, that would otherwise cause the whitespace (either a single space character or a line break) to collapse.